src/relay/backend/build_module.cc

```diff
@@ -307,6 +307,10 @@ class RelayBuildModule : public runtime::ModuleNode {
     }
     pass_seqs.push_back(transform::FoldConstant());
 
+    // Fast math optimizations.
+    pass_seqs.push_back(transform::FastMath());
+    pass_seqs.push_back(transform::FoldConstant());
```

Comment:
We should create a SequentialPass instead of doing this back-to-back. For example, you can return `Sequential({FastMathPass, FoldConstant()}, "FastMath")` in fast_math.cc, where `FastMathPass = CreateFunctionPass(....)`

Reply:
OR I can just move the FastMath before the previous FoldConstant.

I don't think FastMath brings any new const in the graph. I just put it there because FastMath becomes the last pass before FuseOps, and we want the graph to be fully optimized.

Reply:
Yeah, then we probably want to move it after alteroplayout.